src/psrsearch.jl

```diff
@@ -159,12 +155,12 @@ function z_n_search(
     t0 = first(times)
     t1 = last(times)
     df = 1 / (t1 - t0) / oversample
-    freqs = [fmin:df:fmax;]
+    freqs = fmin:df:fmax
```

Comment:
As mentioned above, the range is compact and efficient (you don't store the whole array): unless you need to iterate over it many many times, it's usually better than converting it to an array